### PR TITLE
Update to Electron 1.4.3

### PR DESCRIPTION
**Pull request by miniak**
case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

File: github-electron/github-electron.d.ts [hunk 1]
```diff
@@ -3831,12 +3853,21 @@ declare namespace Electron {
 		 * By default a new BrowserWindow will be created for the url.
 		 *
 		 * Calling event.preventDefault() will prevent creating new windows.
+		 * In such case, the event.newGuest may be set with a reference
+		 * to a BrowserWindow instance to make it used by the Electron's runtime.
 		 */
-		on(event: 'new-window', listener: (event: Event,
+		on(event: 'new-window', listener: (event: Event & { newGuest?: BrowserWindow },
```
Comment (rhysd) [inline github-electron/github-electron.d.ts]: Type definition of `event` looks adhoc.  Should we define new type extending `Event`?

```typescript
interface NewWindowListenerEvent extends Event {
    newGuest?: BrowserWindow;
}
```

Comment (miniak) [inline github-electron/github-electron.d.ts]: done

File: electron/github-electron.d.ts [hunk 2]
```diff
@@ -2655,6 +2655,28 @@ declare namespace Electron {
...
+		/**
+		 * @param rect The area of the image to crop
+		 * @returns The cropped image.
+		 */
+		crop(rect: Rectangle): NativeImage;
+		/**
+		 * @returns The resized image.
+		 * If only the height or the width are specified then the current aspect ratio will be preserved in the resized image.
+		 */
+		resize(options: {
```
Comment (rhysd) [inline electron/github-electron.d.ts]: Is `options` mandatory?  If it can be omitted, `options?` is more  suitable.

**Review by rhysd — CHANGES_REQUESTED** [review]

File: electron/github-electron.d.ts (hunk 2 above)
Comment (miniak) [inline electron/github-electron.d.ts]: https://github.com/electron/electron/blob/master/docs/api/native-image.md#imageresizeoptions

According to the documentation it's not optional.

Comment (rhysd) [inline electron/github-electron.d.ts]: Oh, I see. Thanks.

**Comment by rhysd** [comment]
LGTM :+1:

**Pull request merged**